Conceptpower+Spring/src/integration-test/java/edu/asu/conceptpower/rest/ConceptLookupIT.java

```diff
@@ -0,0 +1,100 @@
+                .andExpect(jsonPath("$.conceptEntries[1].conceptList", is("WordNet")))
+                .andExpect(jsonPath("$.conceptEntries[1].deleted", is(false)))
+                .andExpect(jsonPath("$.conceptEntries[1].concept_uri",
+                        is("http://www.digitalhps.org/concepts/WID-10954498-N-01-Einstein")))
+                .andExpect(jsonPath("$.conceptEntries[1].synonym_ids", is("WID-10954498-N-02-Albert_Einstein,")))
+                .andExpect(jsonPath("$.conceptEntries[1].wordnet_id", is("WID-10954498-N-01-Einstein")))
+                .andExpect(jsonPath("$.conceptEntries[2].id", is("WID-05875723-N-01-Bose-Einstein_statistics")))
+                .andExpect(jsonPath("$.conceptEntries[2].lemma", is("bose-einstein statistics")))
+                .andExpect(jsonPath("$.conceptEntries[2].pos", is("noun")))
+                .andExpect(jsonPath("$.conceptEntries[2].description",
+                        is("(physics) statistical law obeyed by a system of particles whose wave function is not changed when two particles are interchanged (the Pauli exclusion principle does not apply)")))
+                .andExpect(jsonPath("$.conceptEntries[2].conceptList", is("WordNet")))
+                .andExpect(jsonPath("$.conceptEntries[2].deleted", is(false)))
+                .andExpect(jsonPath("$.conceptEntries[2].concept_uri",
+                        is("http://www.digitalhps.org/concepts/WID-05875723-N-01-Bose-Einstein_statistics")))
+                .andExpect(jsonPath("$.conceptEntries[2].wordnet_id", is("WID-05875723-N-01-Bose-Einstein_statistics")))
+                .andExpect(jsonPath("$.conceptEntries[3].id", is("WID-10126926-N-05-Einstein")))
+                .andExpect(jsonPath("$.conceptEntries[3].lemma", is("einstein")))
+                .andExpect(jsonPath("$.conceptEntries[3].pos", is("noun")))
+                .andExpect(jsonPath("$.conceptEntries[3].description",
+                        is("someone who has exceptional intellectual ability and originality; \"Mozart was a child genius\"; \"he's smart but he's no Einstein\"")))
+                .andExpect(jsonPath("$.conceptEntries[3].conceptList", is("WordNet")))
+                .andExpect(jsonPath("$.conceptEntries[3].deleted", is(false)))
+                .andExpect(jsonPath("$.conceptEntries[3].concept_uri",
+                        is("http://www.digitalhps.org/concepts/WID-10126926-N-05-Einstein")))
+                .andExpect(jsonPath("$.conceptEntries[3].synonym_ids",
+                        is("WID-10126926-N-01-genius,WID-10126926-N-02-mastermind,WID-10126926-N-03-brain,WID-10126926-N-04-brainiac,")))
+                .andExpect(jsonPath("$.conceptEntries[3].wordnet_id", is("WID-10126926-N-05-Einstein")))
+                .andExpect(status().isOk());
```

Comment:
Can't you compare the output to a string stored in a file? Would make it easier to maintain and easier to read.